### ISSUE TITLE
Brave crashes with an `invalid-read` error trying to import Wikipedia page via IPFS

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#16425]
## Description 

Brave crashes with an `invalid-read` error trying to import Wikipedia page via IPFS

## Steps to Reproduce

   1. new profile
   2. launch Brave using `--use-dev-goupdater-url`
   3. load `ipns://brantly.eth`
   4. click on `Use a local node`
   5. load `wikipedia.org`
   6. context-click the page and choose `Import to IPFS` > `This page`
   7. once it imports, go to `brave://ipfs-internals/` and click on `My Node`
   8. find the wikipedia import and delete it
   9. repeat steps 5-8 until you crash

## Actual result:

Brave crashes with an `invalid-read` error

https://share.backtrace.io/api/share/hzQQ7egVyUGRtOXPSdFcR0

Callstack (clean exit):
```
[ 00 ] 0x1151a190c
[ 01 ] 0x1130396fa
[ 02 ] 0x11303bda0
[ 03 ] 0x11303be2e
[ 04 ] 0x11303bf17
[ 05 ] 0x1154c3196
[ 06 ] 0x1154c33b9
[ 07 ] 0x115495632
[ 08 ] 0x1154a9bfe
[ 09 ] 0x1154a9911
[ 10 ] 0x11550e554
[ 11 ] 0x1154aa215
[ 12 ] 0x1154754ab
[ 13 ] 0x1154c6288
[ 14 ] 0x1127e9520
[ 15 ] 0x1154c6422
[ 16 ] 0x1154e5328
[ 17 ] 0x7fff203618fc
[ 18 ] 0x7fff2035d443
[ 19 ] 0x1154e52c0
```

## Expected result:
No crash

## Reproduces how often: 

100%

## Brave version (brave://version info)

Brave | 1.27.63 Chromium: 91.0.4472.101&nbsp;(Official Build)&nbsp;nightly&nbsp;(x86_64)
-- | --
Revision | af52a90bf87030dd1523486a1cd3ae25c5d76c9b-refs/branch-heads/4472@{#1462}
OS | macOS Version 11.4 (Build 20F71)

**Comment by spylogsster** [comment]
Unable to reproduce because webui doesnt work for me on 0.9.0 cc @lidel
![image](https://user-images.githubusercontent.com/2965009/122051603-75122a00-cded-11eb-9b33-cbe3e1de1c5b.png)

**Comment by lidel** [comment]
@spylogsster I believe this was fixed in https://github.com/ipfs/ipfs-webui/pull/1790 will try to bubble up this fix to be included in 0.9.
For now, use "Remove" from the context menu behind right-click on a file:
> 
![2021-06-15--14-43-20](https://user-images.githubusercontent.com/157609/122054526-0da5ab80-cde8-11eb-83dd-1e341f793547.png)

**Comment by stephendonner** [comment]
Verified `FIXED` using 


Brave | 1.27.73 Chromium: 91.0.4472.114 (Official Build) beta (x86_64)
-- | --
Revision | 4bb19460e8d88c3446b360b0df8fd991fee49c0b-refs/branch-heads/4472@{#1496}
OS | macOS Version 11.4 (Build 20F71)

Confirmed I was able to import, delete, and repeat both steps, without crashing, multiple times.

Steps:

1. new profile
2. launch Brave using `--use-dev-goupdater-url`
3. load `ipns://brantly.eth`
4. click on `Use a local node`
5. load `wikipedia.org`
6. context-click the page and choose `Import to IPFS > This page`
7. once it imports, go to `brave://ipfs-internals/` and click on `My Node`
8. find the wikipedia import and delete it
9. repeated steps 5-8 without crashing
